docs/sources/configure-client/trace-span-profiles/java-span-profiles.md

```diff
@@ -40,49 +40,52 @@ Your applications must be instrumented for profiling and tracing before you can
 
 ## Configure the otel-profiling-java package
 
-To start collecting Span Profiles for your Java application, you need to include [otel-profiling-java](https://github.com/pyroscope-io/otel-profiling-java) in your code.
+To start collecting Span Profiles for your Java application, you need to include [otel-profiling-java](https://github.com/pyroscope-io/otel-profiling-java) as an extension to your application.
 
-This package is a `TracerProvider` implementation, that labels profiling data with span IDs which makes it possible to query for span-specific profiling data in Grafana Tempo UI.
+Let's assume you have this sample application docker image:
```

Comment:
Docker website uses a capital D when referring to a Docker image. Also, we [avoid using first person](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) (i, me, we, us) in documentation, except for release notes. However, there are cases where things like "Let's" sounds better than rewording it. 
```suggestion
Let's assume you have this sample application Docker image:
```

Reply:
```suggestion
Assuming you have this sample application Docker image:
```

Reply:
This what I went with. Thanks for the feedback ❤️